sdk/src/test/java/com/google/cloud/dataflow/sdk/util/BigQueryTableRowIteratorTest.java

```diff
@@ -125,7 +126,8 @@ private static Table tableWithBasicSchema() {
                 .setFields(
                     Arrays.asList(
                         new TableFieldSchema().setName("name").setType("STRING"),
-                        new TableFieldSchema().setName("answer").setType("INTEGER"))));
+                        new TableFieldSchema().setName("answer").setType("INTEGER"),
```

Comment:
Just add the additional field to testQueryWithNoTable instead of adding another schema